.ci/updatecli/bump-java-version.yml

```
@@ -0,0 +1,46 @@
+---
+name: Update java version file
+pipelineid: "logstash/version-updates-{{ requiredEnv "LOGSTASH_BRANCH" }}"
```

IIUC, you will run this updatecli manifest per active branch (9.1, 8.19, main, ...)


```suggestion
pipelineid: "logstash/jdk-version-updates-{{ requiredEnv "LOGSTASH_BRANCH" }}"
```